frontend/src/components/Agreements/AgreementTypes.d.ts

```diff
@@ -0,0 +1,46 @@
+import { SafeUser } from "../Users/UserTypes";
+import { Project } from "../Projects/ProjectTypes";
+import { BudgetLine } from "../BudgetLineItems/BudgetLineTypes";
+
+export type Agreement = {
+    id: number;
+    name: string;
+    agreement_type: string;
+    description: string;
+    product_service_code_id: number;
+    agreement_reason: string;
+    incumbent: string;
```

Comment:
Can you remove Incumbent? I'm removing it in my own PR so feels like it would make sense to just not include it in the type considering our PRs are going to be going in at similar times.

Reply:
should we replace `incumbent` with `vendor`?

Reply:
done